emscripten-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest-pyodide>=0.54.0
+pyodide-build>=0.24.0
+webdriver-manager>=4.0.0
```

Comment:
We tend to pin instead of using lower bounds. This sounds especially important as the pyiodide dependencies are still in the 0.x range. What do you think?

Reply:
Yeah, pinning should be fine, given urllib3 its a pure python module it's only a testing thing, and I don't think pyodide is likely to break any of this in the short term at least.